src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+  height: fit-content;
+  margin-right: 50px;
```

Comment:
More correctly to use side padding for header, intstead of using margin left/right for nav/logo. Should be smth like this:
![image](https://github.com/mate-academy/layout_catalog/assets/54936044/1280f5c1-9810-4c7c-8e84-daea8e9bf2ac)